src/components/Dashboard/AddressListSm.vue

```diff
@@ -1,16 +1,23 @@
 <template>
   <div id="address-list-sm">
-    <v-expansion-panels accordion multiple :value=[0]>
+    <!-- when grayed out, expand all panels and disable expansion -->
+    <v-expansion-panels accordion multiple
+      :value="(showGrayedOut || showCompleteYourFilingMessage) ? [0,1] : [0]"
+      :disabled="(showGrayedOut || showCompleteYourFilingMessage)"
+    >
       <!-- Registered Office -->
       <v-expansion-panel id="registered-office-panel"
         class="align-items-top"
-        :class="{ 'address-overlay': coaPending }"
+        :class="{
+          'address-overlay': coaPending,
+          'disabled': showGrayedOut
```

Comment:
Cool! Does the CoaPending state still work when a bcomps coa is pending? 😆  I only ask, because there is a ticket remaining to look into that :)

Reply:
BCOMP FE COA filings _should_ work. They are mutually exclusive with IA filings. That said, I haven't tested them explicitly. As I refactored much of the layout, it may be possible to improve their display as well (ie, button and text alignment).

Reply:
Excellent, will give it a quick revisit in the follow up ticket, sounds promising though :)